<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplaceDialog</class>
 <widget class="QDialog" name="ReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>277</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Find and replace</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="replaceWhatLabel">
     <property name="text">
      <string>Replace what:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="replaceWhatEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="replaceWithLabel">
     <property name="text">
      <string>Replace with:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="replaceWithEdit"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="matchCase">
     <property name="text">
      <string>Match case</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="matchWholeWords">
     <property name="text">
      <string>Match whole words only</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="useRegExp">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Use regular expressions</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="searchBackwards">
     <property name="text">
      <string>Search backwards</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="replaceButton">
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replaceAllButton">
       <property name="text">
        <string>Replace all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>replaceButton</sender>
   <signal>clicked()</signal>
   <receiver>ReplaceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>replaceAllButton</sender>
   <signal>clicked()</signal>
   <receiver>ReplaceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
